ocaml/testsuite/tests/typing-layouts/parsing_alpha.compilers.reference

```diff
@@ -7,7 +7,7 @@ Error: This type ('a : value) should be an instance of type ('a0 : void)
        The layout of 'a is void, because
          of the annotation on 'a in the declaration of the type t0.
        But the layout of 'a must overlap with value, because
-         a type argument defaults to layout value.
+         the type argument of imported type list has layout value.
```

Comment:
Actually, after reading all these messages, I think we can drop "imported". And "type". So this would become

```suggestion
         the type argument of list has layout value.
```

Shorter. Simpler.